src/translations/client.csv

```diff
@@ -156,134 +156,23 @@ changeEmail.validation.msg,Phone number validation message,Must be a valid email
-config.informantNotification.success,Notification for informant update success,Informant notifications updated,Mise à jour des notifications des informateurs
-config.informantNotification.title,The title for Informant notifications,Informant notifications,Notifications d'informateurs
-config.integrations,,Integrations,Intégrations
-config.listDetails,Details for certificates templates list,To learn how to edit an SVG and upload a certificate to suite your country requirements please refer to this detailed guide. ,"Pour savoir comment modifier un SVG et télécharger un certificat en fonction des exigences de votre pays, veuillez consulter ce guide détaillé."
-config.listDetailsQsn,Details question for certificates templates list,How to configure a certificate?,Comment configurer un certificat ?
-config.listTitle,Title for certificates templates list,Certification,La certification
-config.marriageDefaultTempDesc,Label for default marriage certificate template,Default marriage certificate template,Modèle de certificat de mariage par défaut
-config.marriageTemplate,Label for marriage certificate template,Marriage certificate,Certificat de mariage
-config.previewTemplate,,Preview,Prévisualiser
-config.printTemplate,Print action in certificate config action menu,Print,Imprimer
-config.uploadTemplate,Upload action in certificate config action menu,Upload,Télécharger
 config.userRoles.language,Language name,"{language, select, en {English} fr {French} other {{language}}}","{language, select, en {Anglais} fr {Français} other {{language}}}"
-config.userRoles.role,ListViewSimplified header for role,ROLE,RÔLE
 config.userRoles.roleUpdateInstruction,Instruction for adding/updating role in role management modal,Add the roles to be assigned the system role of {systemRole},Ajoutez les rôles auxquels attribuer le rôle système de {systemRole}
-config.userRoles.subtitle,Subtile for informant sms notification,Map user roles to each system role so that specific permissions and privileges are correctly assigned. To learn more about the different system roles see ... {link},"Associez les rôles d'utilisateur à chaque rôle système afin que les autorisations et les privilèges spécifiques soient correctement attribués. Pour en savoir plus sur les différents rôles système, voir ... {link}"
-config.userRoles.systemRoleSuccessMsg,Label for System role updated success message,System role updated successfully,Rôle système mis à jour avec succès
-config.userRoles.systemRoles,ListViewSimplified header for system roles,SYSTEM ROLES,RÔLES SYSTÈME
-config.userRoles.title,The title for user roles,User roles,Rôles des utilisateurs
```

Comment:
tip for review: can use (by use, i mean to tweak the script) the `extract-translation` script